msmarco-passage-ranking/challenges/default.json

```diff
@@ -70,14 +70,40 @@
       "operation": "text-expansion-splade-search-maxwand-enabled",
       "warmup-iterations": 100,
       "iterations": 1000,
-      "clients": {{search_client | default(1)}}
-  },
+      "clients": {{search_clients | default(1)}}
+    },
     {
       "operation": "text-expansion-splade-search-maxwand-disabled",
       "warmup-iterations": 100,
       "iterations": 1000,
       "clients": {{search_clients | default(1)}}
-  },
+    },
+    {
+      "operation": "pruned-text-expansion-search-maxwand-enabled",
```

Comment:
I suggest adding a "name" field besides "operation" field. This is helpful if you want to include/exclude a specific task while running benchmark. [This](https://github.com/elastic/rally-tracks/blob/master/wikipedia/challenges/default.json#L41-L42) is an example.

Reply:
Good call, adding!